assets/js/custom/KeepOrDiscardDialog.js

```diff
@@ -0,0 +1,24 @@
+import Swal from 'sweetalert2'
+
+export class KeepOrDiscardDialog {
+  constructor (saveOnLanguageChangeText, btnDiscardText, keepText) {
+    this.saveOnLanguageChangeText = saveOnLanguageChangeText
+    this.btnDiscardText = btnDiscardText
+    this.keepText = keepText
+  }
+
+  show (callback) {
+    Swal.fire({
+      title: this.saveOnLanguageChangeText,
+      icon: 'question',
+      showDenyButton: true,
+      showCloseButton: true,
+      allowOutsideClick: true,
+      backdrop: true,
+      confirmButtonText: this.keepText,
+      denyButtonText: this.btnDiscardText
+    }).then((result) => {
```

Comment:
Same here

Reply:
This doesn't seem to be working when you have to pass back a parameter. It says it is undefined. Is there another way I could do it or is this okay?

Reply:
fine

Reply:
`then(callback)` will work

Reply:
Updated to this 👍